eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -99,6 +98,17 @@ jobs:
       - src/coreclr/vm/*
       exclude:
       - '*'
+    - subset: wasmbuildtests
+      include:
+      - src/tasks/*
+      - src/tests/BuildWasmApps/*
+      - src/mono/wasm/build/*
+      - src/mono/wasm/runtime/*
+      - src/mono/nuget/*
+      - src/mono/mono/*
+      exclude:
+      - src/mono/nuget/Microsoft.NET.Runtime.Android.Sample.Mono/*
+      - src/mono/nuget/Microsoft.NET.Runtime.iOS.Sample.Mono/*
```

Comment:
nit: excluding like this feels pretty brittle since it'll "break" once we add new platforms. I'd prefer to `include` 
only the specific wasm directories in `src/mono/nuget/` instead.